plugins/modules/site_workflow_manager.py

```diff
@@ -117,20 +121,39 @@
                 description: Floor number within the building site (e.g., 5). This value can only be specified during the creation of the
                     floor and cannot be modified afterward.
                 type: int
-
+              bulk_operation:
```

Comment:
Why do we use a string type for bulk_operation? Can't we use boolean instead and set it to false by default? Also bulk operations can be done only in new versions. We need to document that and take care in the code..
What should be the default value? True or False in latest version?

Reply:
Added bulk_operation as a boolean